db/migrate/20191031090746_add_normalized_domain_to_stories.rb

```diff
@@ -0,0 +1,5 @@
+class AddNormalizedDomainToStories < ActiveRecord::Migration[5.2]
+  def change
+    add_column :stories, :normalized_domain, :string
```

Comment:
Please name this `domain` and have the migration store the domain for all stories and then set the field to be `not null`. That'll let us run queries, remove the complexity of the `domain` method and instance variable, etc.